getting-started/README.md

```diff
@@ -0,0 +1,20 @@
+# Getting Started
+
+The documentation for this example can be found at the following link: https://quarkus.io/guides/getting-started-guide
+
+### Build and Run
+
+To build and run the `getting-started` example, run the following using `Maven`
+or `Gradle` at the root of this directory to start the application:
+
+**Maven**
+
+```
+./mvnw compile quarkus:dev
+```
+
+**Gradle**
+
+```
+./gradlew quarkusDev
+```
```

Comment:
We don't really want this example to be built as a Gradle project. We have a PR separating the projects, I need to take a look at them.

I don't know if it's a good idea to add this file in each quickstart. I wonder if it won't be a nightmare to maintain.

Reply:
I can agree to the issue of maintaining all these. Is there a general pattern to building all of these examples to just get them running? 

Should `./mvnw compile quarkus:dev` be enough to get everything up and running, or are there certain things that some examples will need in addition?